src/instruction.cpp

```diff
@@ -406,6 +406,9 @@ void instruction::print(std::ostream& os,
     // skip return instruction shape
     if(ins->name() != "@return")
         os << " -> " << ins->get_shape();
+    // print tid
+
+    os << ", tid=" << ins->target_id;
```

Comment:
Can we just expand it so there's no ambiguity?
```suggestion
    os << ", target_id=" << ins->target_id;
```